tokio/build.rs

```diff
@@ -117,6 +161,14 @@ fn main() {
         autocfg::emit("tokio_no_const_mutex_new")
     }
 
+    if !target_has_atomic_u64 {
+        // To disable this feature on compilers that support it, you can
+        // explicitly pass this flag with the following environment variable:
+        //
+        // RUSTFLAGS="--cfg tokio_no_atomic_u64"
+        autocfg::emit("tokio_no_atomic_u64")
+    }
```

Comment:
This variable naming is a bit confusing currently, because `target_has_atomic_u64` is always true on the latest compiler even if the target doesn't have atomic u64.

Reply:
That's certainly an option.